src/wp-includes/default-filters.php

```diff
@@ -328,6 +328,7 @@
 add_action( 'wp_head', 'wp_shortlink_wp_head', 10, 0 );
 add_action( 'wp_head', 'wp_custom_css_cb', 101 );
 add_action( 'wp_head', 'wp_site_icon', 99 );
+add_action( 'wp_footer', 'wp_enqueue_global_styles', 1 );
```

Comment:
Can we move this next to the other action hook for `wp_enqueue_global_styles` so it's obvious that there's two of them?

Reply:
+1 for this. They should be together to improve readability/understanding

Reply:
Moved and added an inline doc note.